neofetch

```diff
@@ -317,7 +317,7 @@ get_uptime() {
                     boot="${boot/,*}"
 
                     # Get current date in seconds
-                    now="$(date +%s)"
+                    now="$(printf "%(%s)T")"
```

Comment:
Does this line work on macOS @iandrewt? Bash has builtin support for all date's format options with the caveat that you can only output the current date. (You can't get a previous date's info)